lib/statsig_logging.ex

```diff
@@ -0,0 +1,123 @@
+defmodule Statsig.Logging do
+  use GenServer
+  require Logger
+
+  @type state :: %{
+    flush_interval: integer() | nil,
+    events: list(),
+    flush_timer: reference() | nil
+  }
+
+  def start_link(_) do
+    GenServer.start_link(__MODULE__, %{
+      flush_interval: 60_000,
+      events: [],
+      flush_timer: nil,
+    }, name: __MODULE__)
+  end
+
+  def initialize(flush_interval) do
```

Comment:
what's the intention behind the initialize functions?

Reply:
Hmm I needed it for how I set up my test framework, but I think I can change that to accomodate